ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/services/SandboxClientResource.scala

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.services
+
+import java.net.{InetAddress, InetSocketAddress}
+import java.util.concurrent.TimeUnit
+
+import com.digitalasset.ledger.api.testing.utils.Resource
+import io.grpc.netty.NettyChannelBuilder
+import io.grpc.{Channel, ManagedChannel}
+import io.netty.channel.EventLoopGroup
+import io.netty.channel.nio.NioEventLoopGroup
+import io.netty.util.concurrent.DefaultThreadFactory
+
+class SandboxClientResource(port: Int) extends Resource[Channel] {
```

Comment:
Probably not necessary but it looks like `eventLoopGroup` and `channel` are probably good candidates to be wrapped in `Resource`s.

Reply:
Agreed, and they are in production. I was rushed for time.

Reply:
I see, thanks.